enemies.js

```diff
@@ -238,4 +238,78 @@ class PirahnaPlant {
             }
         }
     };
-};
\ No newline at end of file
+};
+
+class FireBar {
+    constructor(game, x, y, numOfFires) {
+        Object.assign(this, { game, x, y , numOfFires});
+        let brick = new Brick(this.game, this.x, this.y, 3, "None");
+        this.game.addEntity(brick);
+
+        this.angle = 0;
+        this.fires = [];
+        for (var i = 0; i < this.numOfFires; i++) {
+            var x = i * 25 * Math.sin(this.angle);
+            var y = i * 25 * Math.cos(this.angle);
+            if(i <= 1) {
+                var fire = new FireBar_Fire(this.game, this.x + x + 12, this.y + y + 12, true);
+            } else {
+                var fire = new FireBar_Fire(this.game, this.x + x + 12, this.y + y + 12, false);
+            }
+            this.fires.push(fire);
+            this.game.addEntity(fire);
+        }
+    };
+
+    update() {
+        this.angle += this.game.clockTick;
+        if(this.angle >= 360){
+            this.angle = 0;
+        }
+        console.log(this.numOfFires);
```

Comment:
this needs to go before final implementation irrc. but even if our project does not have the "no console outputs" policy I hope you know that its considered bad practice to leave console-outputs in most situations....especially in a method that gets called as much as update().

I strongly recommend that you, at the least, wrap such statements in a "if debug" check.